nengo/spa/input.py

```diff
@@ -40,7 +40,10 @@ def on_add(self, spa):
         Module.on_add(self, spa)
 
         for name, value in iteritems(self.kwargs):
-            target, vocab = spa.get_module_input(name)
+            try:
+                target, vocab = spa.get_module_input(name)
+            except TypeError:
+                raise ValueError('Could not find input called "%s"' % name)
```

Comment:
And if we do this conversion, we should probably do it in `spa.get_module_input` to have a consistent behaviour.